test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/AuthorizationApplicationModelProviderTest.cs

```diff
@@ -116,6 +117,40 @@ public void CreateControllerModelAndActionModel_AllowAnonymousAttributeAddsAllow
         }
 
         [Fact]
+        public void CreateControllerModelAndActionModel_UseDefaultAuthorizationPolicyProvider()
```

Comment:
Ideally, the name should say something about what's being tested. `WhenIDoSomething_AndSomethingIsTrue_AThingHappens()`

A better name would be `OnProvidersExecuting_DefaultPolicyProvider_NoAuthorizationData_NoFilterCreated()`